apps/src/templates/watchers/Watchers.jsx

```diff
@@ -411,6 +411,7 @@ class Watchers extends React.Component {
                   attachment: 'together'
                 }
               ]}
+              style={{zIndex: 2}}
```

Comment:
Should we put this into `styles`?  It will also give us a chance to name this styling which might give more of a hint why it's useful.

Reply:
Oh, I actually meant to the `styles` object that we seem to be using, e.g. at https://github.com/code-dot-org/code-dot-org/pull/32388/files/4d8f73a84f401f8a27109912a16330e982c45544..7c8c2b09718c4a9f2ead431838a9bdd695d2c96c#diff-8f69bd0a5172bb6180dcdc17fb86ec35R402